acceptance/.gitattributes

```diff
@@ -0,0 +1,5 @@
+# We should always treat new lines in acceptance tests in windows as \n.
+# This is important because the characters itself can be semantically important in tests.
+# For example, an additional \r character in windows changes the upload payload if you are
+# uploading the file's content to a workspace.
+* text eol=lf
```

Comment:
There are binary files in this tree (`.whl` files, for example).

This rule should only apply to `.txt` files (and others if we find they exhibit the same problem).

Reply:
Good catch, thanks! Only applying the annotation to .txt files.